File: backend/secondbrain/src/main/java/uknowklp/secondbrain/api/auth/dto/GoogleAuthRequest.java
package uknowklp.secondbrain.api.auth.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;

// 모바일 앱에서 Google ID Token을 받는 요청 DTO
// 웹은 OAuth2 플로우를 사용하지만, 모바일은 Google Sign-In SDK를 사용하므로 별도 엔드포인트 필요
@Getter
@NoArgsConstructor
public class GoogleAuthRequest {
	private String idToken; // Google Sign-In SDK에서 받은 ID Token
}

File: mobile_watch/mobile/src/main/java/com/example/secondbrain/data/model/GoogleAuthRequest.kt
package com.example.secondbrain.data.model

import com.google.gson.annotations.SerializedName

// Google ID Token을 백엔드로 전송하기 위한 요청 모델
data class GoogleAuthRequest(
    @SerializedName("idToken")
    val idToken: String
)
